rust-toolchain

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "stable"
```

Comment:
I do not think this is bringing much to the table.

My understanding is that the toolchain is mostly useful for pinning specific nightly revision.

What if the user has not pulled the latest stable? It could still fail without the clear information of the MSRV.

TBH I am not convinced this change is improving on the status quo.

Reply:
I am afraid i am not follow, do we come to a agreement the msrv is not need. We just want always use the latest stable?

Reply:
Yes, we don't want to pin to some MSRV and just use stable.

The `rust-toolchain` file you added doesn't guarantee that we use the latest nightly.

Reply:
Latest stable, sorry!

Reply:
So you mean we should pin to a stable/nighty version, after the latest stable released, run some tests, if everything work as expected, upgrade to that version?

Reply:
No. I mean that pinning `"stable"` here does not make much sense. It doesn't refer to the latest released stable per-se.

Reply:
To clarify, just removing `rust-version = ""` from `Cargo.toml` is probably enough.

Reply:
> No. I mean that pinning `"stable"` here does not make much sense. It doesn't refer to the latest released stable per-se.

Sorry, I am confused, if not what is the value for the **stable**

Reply:
Thx for the explanation, though already closed 😶